modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java

```diff
@@ -97,6 +100,8 @@ public class EmailNotificationTypeTest extends BaseNotificationTypeTest {
 	public static void setUpClass() throws Exception {
 		BaseNotificationTypeTest.setUpClass();
 
+		_group = GroupTestUtil.addGroup();
```

Comment:
@rijema this variable should be after `_freeMarkerTermValues`, please sort it.

Thank you!